src/common/components/i18n/i18n.tsx

```diff
@@ -1,4 +1,6 @@
-/*
+export const DO_NOTHING = '';
+
+/* We keep that for future reuse. Internationalization has been postponed.
```

Comment:
Why are we still keeping this file???

Reply:
Right, we can remove it.
It was commented-out, proposed change is to keep existing state and making it just compilable.